src/Admin/AbstractAdmin.php

```diff
@@ -3261,6 +3251,14 @@ public function canAccessObject($action, $object)
         return $this->hasAccess($action, $object);
     }
 
+    /**
+     * @return MutableTemplateRegistryInterface
+     */
+    final public function getTemplateRegistry()
```

Comment:
Why did you make this public?

Reply:
To resolve this issue:
https://github.com/sonata-project/SonataAdminBundle/pull/6383#discussion_r487426269

If we dont wanna grant access to setter we should these methods behind decorator patter. Access to templates should be easy.